adg/pipelines/scripts/coverage-report.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+if [ $(ls reports/coverage*.xml | wc -l) != "7" ]; then
+    echo "There are some missing coverage files."
+    echo "Existings files are:"
+    ls reports/coverage*.xml
+    exit
+fi
+
+set -e
+set -o pipefail
+
+FAIL_UNDER=80
+
+coverage xml --rcfile=.coveragerc-local
+coverage html --rcfile=.coveragerc-local
+
+echo "*** Generating diff-cover report ***"
+ls reports/coverage*.xml
```

Comment:
Command has been duplicated multiple times. Let's reuse it via a variable

Reply:
Same, We are not using this file for now, Will be updated in a new story.